go.mod

```diff
@@ -47,23 +47,20 @@ require (
 	go.uber.org/multierr v1.1.0 // indirect
 	go.uber.org/zap v1.10.0 // indirect
 	golang.org/x/crypto v0.0.0-20190829043050-9756ffdc2472 // indirect
-	golang.org/x/lint v0.0.0-20190409202823-959b441ac422 // indirect
 	golang.org/x/net v0.0.0-20190827160401-ba9fcec4b297 // indirect
 	golang.org/x/sys v0.0.0-20190904005037-43c01164e931 // indirect
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
-	golang.org/x/tools v0.0.0-20190903163617-be0da057c5e3 // indirect
-	google.golang.org/appengine v1.6.2 // indirect
 	google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55 // indirect
 	google.golang.org/grpc v1.23.0
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	gopkg.in/gcfg.v1 v1.2.3
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
-	honnef.co/go/tools v0.0.1-2019.2.2 // indirect
 	k8s.io/api v0.0.0-20190831074750-7364b6bdad65
 	k8s.io/apimachinery v0.0.0-20190831074630-461753078381
 	k8s.io/client-go v0.0.0-20190831074946-3fe2abece89e
 	k8s.io/klog v0.4.0
-	k8s.io/sample-controller v0.0.0-20190831080103-b17b22266fdc
+	k8s.io/kubernetes v1.11.2
```

Comment:
I'll file an issue to follow-up on this later, but this is a *really* old version of k/k to pull in. It's outside of the "n-2" support policy. The vSphere CPI recently updated it's dependency to 1.15.0, and I know it was a ton of work to do so. At some point the same thing should happen here.

Reply:
And actually, as I've looked at the new *.go files, the only direct imports of `k8s.io.*` packages are of `klog`, `client-go`, and `api`. So I'm wondering why `kubernetes` is being added as a direct dependency here at all... 🤔

Reply:
Good catch! I had some tests as part of this PR earlier and overlooked removing it when i removed the tests. Since you've pointed it out, maybe we'll look at that internally and try to resolve any issues ASAP. Thanks!